src/request.js

```diff
@@ -0,0 +1,18 @@
+/* eslint-disable no-console */
+'use strict';
+
+const http = require('http');
+
+const options = {
+  hostname: 'localhost',
+  port: 3006,
+  // path: '/file/index.html',
```

Comment:
Remove comment. Try it write obvious code that explain all we need without comments)
Fix all cases